src/app/parcels/ActionBar/ActionModals/DriverOverviewModal.tsx

```diff
@@ -87,12 +91,17 @@ const DriverOverviewModal: React.FC<ActionModalProps> = (props) => {
     const onClose = (): void => {
         props.onClose();
         setDate(dayjs());
-        setDriverName("");
+        setDriverName(null);
         setErrorMessage(null);
     };
 
     const onPdfCreationCompleted = async (): Promise<void> => {
-        const { error } = await props.updateParcelStatuses(props.selectedParcels, props.newStatus);
+        const { error } = await props.updateParcelStatuses(
+            props.selectedParcels,
+            props.newStatus,
+            `with ${driverName ?? "Unknown Driver"}`,
```

Comment:
Isn't the driver name required? If we set it to null maybe we can instead cover cases where driver name is just whitespace

Reply:
The driver name is not required - but agree that it would be good to also use "Unknown Driver" if `driverName` is just whitespace - I think this will be the case if I use `trim` as in your other comment

Reply:
Sorry, completely ignore me, the driver name is required

Reply:
Do you want me to make any changes here? since the download button is only enabled when the input is valid (ie the driver name is non null) it should not be possible for this function to be called when the driverName is null, however TS doesn't realise this, it thinks driverName could still be null

Reply:
Does it allow you to delete all of the nulls from `string | null`? We can probably just return everything to the state it was in before and then all the trimming etc issues are going to be covered by the validation ticket you raised

Reply:
You mean change the type of the driverName state back to `string` only and default it to `""`?

Reply:
Yep although I'm happy with the current solution too

Reply:
Ok, I see how the old solution was simpler, but I like the use of null because it represents a driver name that hasn't been entered yet or is invalid and I think that makes more sense to represent as null rather than an empty string

Reply:
(i'll keep it as current solution)